doc/architecture/domains/certificates.md

```diff
@@ -24,23 +24,41 @@ Note: `ContractService` is currently getting information about a metering point
 ![Issuer component diagram](../diagrams/certificates.current.component.certificate.api.drawio.svg)
 
 ### Message flow: Issue certificate
-The sequence diagram below shows the flow of messages between the components when issuing a certificate. All messages are published to the message broker; the message broker is not shown in the diagram.
+The sequence diagram below shows the flow of messages between the components when issuing a single certificate. All messages are published to the message broker; the message broker is not shown in the diagram. Before that flow is possible, a wallet deposit endpoint must be created in the user's wallet; that happens as part of creating the Contract.
```

Comment:
Hvornår er det nu en Contract bliver lavet? Synes ikke, jeg kan finde det på demo.energioprindelse.dk

Reply:
Når en bruger trykker "Activate"-knappen ud for metering points i frontenden. Så bliver der oprettet en kontrakt med start dato = nu og ingen slut-dato.